.travis.yml

```diff
@@ -17,29 +17,36 @@ addons:
+  - secure: N1XUp8mKRJALWVJXoxwXh/yy8tptEUvSEyWQEqU+qK85DGJnen7LwZEDX3BSxLvhc9++vbIw2wVH/3p7tjqAJu1h5+Du/G/gsqLIQhEACvezTTnblGvFONTBg1/ZEUqN3ZJkpLGcOvNWIuk4oDcYejGOekOkj4io7H88ynON4vM4piJfmRcvl1YKy51FwRzfQlfoCIbGVXMOnVn3S4QW29Yc14hNassen/b0Pt7aNlsQaEV8aPboD2+N1K2DF2W1uV75p/4tD3i6YRyMOWEPhenLv+BlFTfdHz1+IORdZiElUMO1z2obe5x8FvpF1EpKmG/6TfGpdYajwZR7v0EaTd/FSEJs0y988dZWZz/HEPlHtHyHyG7yYgxVKtY8Axy6NXu+xCzI1tuUr0sANlaV0oJINJ5mpF6kojR9sjWsOwihEVP8ulGwqn4oYjn7+D1QB4fn+Vq+Qm4YcKpD4DNnvQL/pGWkD1pMH5eqFNjCt0F2i5+uMiKjrbGjOt0e3nSVT5/Y9B00HFRNuHnckdNHS+OyNi+67RtXDokIuV+kMYKL77X/cHKn5NYL6QNIvNJ1/XhMTDOXFtLmirIEdHwSdNh4wuFpEWtgKcEwtzqQd70FzwPy8yRc7kwsKv4HMAQsJEdJeNr/0AFW8VLgN0a3jJ4rZx35KRorwi2ZY77ziss=
+  - secure: Jx9pF/T6RdzjGdQ6KFMF6b8aAcP5mBZDOMXlCmOUZ4xj4GMV52h6+f3DCIk1YRzpGdpmCNeDLlxJ2RQR7YLO7UKX7vGtMIVW1dZd5YBYuVAN3fl85vt3J+w9/qA7fhZDfKzrDXxIHa++WKIGXG5kb1kXPQ6bMRWMnq38sPXWNrQ7FR2jd1B79xP48lTulhkExi4FcSm8u+mHkdZjkLDG8XImruakn4Ax7CzbYxaBqB3xX3Z9F9nUNRcjKQO8Nw/2pWKByOKSsiRIXlrFPq7lsICLVP+e51xamYuEgp1ZjoP54/hjZsqSv09ObTu29a5R80EsEzhjxL2RU58pvJWXG4c7bSaoz6yYgYRAD8dO8z8O4G+yrv9clFbxoE3N/7o3rkBMhP7+OXPzKDMa8ZuC6vaQI23go7Lq+ypSsRbAFthmmsPQJJtGZcgU8m1vbD3wFHusMNuks6yk6rJr61VO3dODACB7OKlFma2wQzOpXHpiT6pIc2jAIlrlSYYe/17O0PnDMjIPmUHsX2Qpg9/2bhhzNjWbuLgdwu+kspBhh/njibQ4LxmcMjIGVsjpCflEGVTwChIqKXFrQScw/4kjTwh2WaEcq26Mgtb7CRj1S2tssr3Ko5v1Kk4sC0JtohaMGXGLwS485zEI/eNWphac8Wi+kmNsQ8omCQYMawM8I1A=
+  - secure: xZyGnuzLFt3Afv0RpLu8d+tFoz0xAn1L9UIbvsy9cH5NL01YqXnZic5+Oh+RhJCpdtZ0GGwCFlPDsEwlJPK3e7m6qmzPveJTgb7ec0C/SWyKIWYE+EG3XDszFEIGxxPW0Zg/RJ5tCuSI9W4YtjPbOkPTL17Mi+NXqkfdvlunBqkxG0GBChADrTfy+5YMAXCHvhQcfG5ULK+PTT+CemKwbAxIpcB1wHsZErKIaBncBmTELQj+ZzuT1Kp9VPgMolwz/2m7hDET7OaBa0ZVJh1hPB/Ffh6W8DrinR+X7z5Bfk5fi79JGdx2ZXwznMXs/ddKZPJOFmA2cHPD9+A7Lvsvwaj4OjkWuq5VAQT5lUxU2FHM1PB8FLBQkzXs/xxnbhX1qT7IuvP9LLQYbCPtIxuffSho8/TxGCtGKfcAoiG0xI3DtGT5/kiOf9hacV0dIY206tT3AnnOhvB36PvtuikyLPT0E2j9AS4Lczl7hAbaJ3ujXVlWKYI4Coz0qO+zRugEKkGRed4LCeg3wq2SO9p5Mz8j5KEirev0VKTrkO9nLm0XezmCq3CZrcNCnmUGI6sxAwjGjGipylx0pC0NfL7qQI/xMruKe+eBuSsLwFJr4fNLtCj1BY9jwtmN85oKAAHyiLJOIxv96HZe+B4tCLqsxVGW6/zyVh345E8Fw9C7XZ4=
 
-matrix:
-  include:
-    - env:
-      - SAUCE_BROWSER_NAME="safari"
-      - SAUCE_BROWSER_VERSION="9.0"
-      - SAUCE_PLATFORM="OS X 10.11"
-    - env:
-      - SAUCE_BROWSER_NAME="internet explorer"
-      - SAUCE_BROWSER_VERSION="10.0"
-      - SAUCE_PLATFORM="Windows 8"
-    - env:
-      - SAUCE_BROWSER_NAME="firefox"
-      - SAUCE_BROWSER_VERSION="latest"
-      - SAUCE_PLATFORM="Windows 10"
+# REMOVE THESE BROWSERS FOR NOW UNTIL WE CAN SOLVE THE 1 LOGIN/MIN/USER PROBLEM.
+# matrix:
+#   include:
+#   - env:
+#     - SAUCE_BROWSER_NAME="safari"
+#     - SAUCE_BROWSER_VERSION="9.0"
+#     - SAUCE_PLATFORM="OS X 10.11"
+#   - env:
+#     - SAUCE_BROWSER_NAME="internet explorer"
+#     - SAUCE_BROWSER_VERSION="10.0"
+#     - SAUCE_PLATFORM="Windows 8"
+#   - env:
+#     - SAUCE_BROWSER_NAME="firefox"
+#     - SAUCE_BROWSER_VERSION="latest"
+#     - SAUCE_PLATFORM="Windows 10"
```

Comment:
No, I figured I'd leave it in as a reminder that we should figure out this rate limiting issue.